docs/doxygen/ngraph_cpp_api.config

```diff
@@ -0,0 +1,18 @@
+@INCLUDE = @IE_CONFIG_BUILD@
+
+EXCLUDE_SYMBOLS        = 
+
+PREDEFINED             =
+
+EXCLUDE_PATTERNS       = */python/*
+
+FILE_PATTERNS          = *.cpp \
```

Comment:
need to add tag file generation and re-use for Plugin API / Ie docs (since ngraph docs refer to ngraph API reference).